include/spdk/bdev.h

```diff
@@ -133,6 +133,7 @@ enum spdk_bdev_io_type {
 	SPDK_BDEV_IO_TYPE_UNMAP,
 	SPDK_BDEV_IO_TYPE_FLUSH,
 	SPDK_BDEV_IO_TYPE_RESET,
+	SPDK_BDEV_IO_TYPE_PASSTHRU,
```

Comment:
This should be SPDK_BDEV_IO_TYPE_NVME_PASSTHRU in case we want to pass through other command sets later.

It also might be a good idea to make I/O and admin commands two separate I/O types so that bdev modules can support just one or the other.